uwrt_mars_rover_drivetrain/uwrt_mars_rover_drivetrain_description/urdf/sensors.macro.xacro

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://wiki.ros.org/xacro">
+
+  <!-- import macros file -->
+  <xacro:include filename="$(find uwrt_mars_rover_drivetrain_description)/urdf/drivetrain.macro.xacro"/>
+
+  <xacro:macro name="generate_sensor_frame" params="sensor_params name">
+    <link name="${name}">
+      <xacro:identical_visual_collision>
+        <identical_blocks>
+          <geometry>
+            <box size="${sensor_params['dim']['length']} ${sensor_params['dim']['width']} ${sensor_params['dim']['height']}"/>
+          </geometry>
+          <material name="blue">
+            <color rgba="0 0 0.8 1.0"/>
+          </material>
+        </identical_blocks>
+      </xacro:identical_visual_collision>
+      <xacro:default_inertial/>
+    </link>
+
+    <joint name="${sensor_params['parent_link']}_to_${name}" type="fixed">
+      <parent link="${sensor_params['parent_link']}"/>
+      <child link="${name}"/>
+      <origin xyz="${sensor_params['offset']['x']}     ${sensor_params['offset']['y']}     ${sensor_params['offset']['z']} " 
+              rpy="${sensor_params['offset']['r_rot']} ${sensor_params['offset']['p_rot']} ${sensor_params['offset']['y_rot']} "/>
+    </joint>
+
+  </xacro:macro>
+
+
+  <xacro:macro name="default_depth_camera_frame" params="camera_link_name">
+    <gazebo reference="${camera_link_name}">
+      <sensor name="${camera_link_name}_sensor" type="depth">
+        <visualize>true</visualize>
+        <update_rate>30.0</update_rate>
+        <camera name="camera">
+          <horizontal_fov>1.047198</horizontal_fov>
+          <image>
+            <width>640</width>
+            <height>480</height>
+            <format>R8G8B8</format>
+          </image>
+          <clip>
+            <near>0.05</near>
+            <far>3</far>
```

Comment:
We should parameterize this stuff in the config yaml file as well so that we can easily tune it later.